blog/page1.md

```diff
@@ -1,54 +1,62 @@
 
-**These are the portions the tutorial will be in:**
+**In this tutorial, we'll be walking through:**
 
 1. Creating the "Upload" page and an HTTP Trigger Function that will upload the user's image to a storage container.
-2. Setting up an Event Grid Subscription and a Function that converts the image into a PDF and stores it again. *The API lives here*
+2. Setting up an Event Grid Subscription and a Function that converts the image into a PDF and stores it again. 
+   - This is where the API will live!
 3. Creating the "Download" page and an HTTP Trigger Function that retrieve the correct PDF. 
-4. **Optional** If you're not lazy, let's add another Function to delete the files and keep our containers sqeauky clean.
+4. **Optional** For those who are interested, we can add another Function to delete the files and keep our containers squeaky clean. 
+   - **Note**: The diagram above excludes the optional deletion feature.
+
+You can find a sample of the final product at [my Github repository](https://github.com/emsesc/bunnimage). 
 
 ### Before we start:
 
-* Make sure you have an [**Azure Subscription**](https://azure.microsoft.com/en-us/free/) so we can utilize the amazing features of Microsoft Azure Functions. (It's free!) 🤩
+* Make sure you have an [**Azure Subscription**](https://azure.microsoft.com/en-us/free/) so we can utilize the amazing features of Microsoft Azure Functions (It's free!) 🤩
+* **Register** for an account on [**Online Convert**](https://www.online-convert.com/register) (with the free version), as we will be using this API convert our images
 * If you want to host your website somewhere, check out [Repl.it](https://repl.it/languages/html), or you can just have your project run [locally](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer)
-* Register for an account on [Online Convert](https://www.online-convert.com/register) (of course with the free version), as we will be using this API convert our images
 
 ## Step 1: Upload the image ⬆️
 
 ### Creating a Function App
-We're going to have a lot of triggers in this project, so we must get started by [creating a Function App](https://docs.microsoft.com/en-us/azure/azure-functions/functions-create-first-azure-function)! Create the Function App and proceed to create the first HTTP trigger (*this will upload our image*).
-
-**Before we start coding the trigger, though, we need to install some npm packages/libraries.**
-![image](https://user-images.githubusercontent.com/69332964/99189070-59e31d00-272d-11eb-80a4-17444e5fac65.png)
+We're going to have a lot of triggers in this project, so let's get started by [creating a Function App](https://docs.microsoft.com/en-us/azure/azure-functions/functions-create-first-azure-function)! Follow those steps to create the Function App, and then create the first HTTP trigger (*this will upload our image*).
```

Comment:
oh also you should note that people should keep track of what their storage account name, and/or resource group name is